### PR TITLE
Dockerfile and scripts for running interop tests

### PR DESCRIPTION
**Pull request by fedinskiy**
### Summary

This is mostly a draft, and maybe we should put these files into separate repo.

Please select the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [x] Refactoring
- [ ] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [ ] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

File: .openshift-interop/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+RUN git clone --depth=1 -b ${QUARKUS_BRANCH} https://github.com/quarkus-qe/quarkus-test-suite.git tests
+WORKDIR /tmp/tests
+
+# maven settings for repository
+ADD https://gitlab.cee.redhat.com/quarkus-qe/quarkus-ocp-pit/-/raw/main/settings.xml settings.xml
+
+ADD --chmod=755 *.sh /tmp/tests/
+
+# test results are in $PROJECT/target/failsafe-reports/*.xml for every PROJECT in $PROJECTS.
+CMD ./oc_login.sh && source "/root/.sdkman/bin/sdkman-init.sh" && ./run.sh
```
Comment (Simranpal) [inline .openshift-interop/Dockerfile:33]: As discussed, the login credentials, kubeconfig will be available in `SHARED_DIR` by an Openshift CI workflow [ipi-aws](https://cspi-qe.gitbook.io/ocp-ci-onboarding/onboarding/onboarding_guide/scenario_development_guide#important-workflows) 
So the `oc_login.sh` is not needed here. We will add that in the Openshift CI config

File: .openshift-interop/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ENV LANGUAGE='en_US:en'
+WORKDIR /tmp
+
+# ubi8 repos contain maven 3.5 and jdk 1.8; we need something newer
+RUN dnf install -y wget git zip unzip --setopt=install_weak_deps=False
+RUN wget -O sdkman.sh https://get.sdkman.io && /bin/bash sdkman.sh
+RUN source "/root/.sdkman/bin/sdkman-init.sh" && sdk install java 22.3.2.r17-mandrel && sdk install maven 3.8.7
+
+# oc client
+ADD https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux-4.13.0.tar.gz oc.tar.gz
```
Comment (rsvoboda) [inline .openshift-interop/Dockerfile:12]: https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.13.0/

File: .openshift-interop/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ENV QUARKUS_PLATFORM_ARTIFACT_ID=quarkus-bom
+
+## List of projects to include in smoke test
+ENV PROJECTS=config,lifecycle-application,http/http-minimum,http/http-minimum-reactive,sql-db/sql-app,monitoring/microprofile-opentracing
+
+RUN git clone --depth=1 -b ${QUARKUS_BRANCH} https://github.com/quarkus-qe/quarkus-test-suite.git tests
+WORKDIR /tmp/tests
+
+# maven settings for repository
+ADD https://gitlab.cee.redhat.com/quarkus-qe/quarkus-ocp-pit/-/raw/main/settings.xml settings.xml
```
Comment (rsvoboda) [inline .openshift-interop/Dockerfile:28]: put the file into the repo / this location

Comment (rsvoboda) [reply .openshift-interop/Dockerfile:28]: Or put RH repo definition into separate profile in pom.xml and activate it in `run.sh`

**Comment by fedinskiy** [comment]
@rsvoboda we need this to be merged for integration with upstream. Any changes you would like to add?

**Review by rsvoboda — APPROVED** [review]
No further comment, let's merge it and proceed with onboarding.

**Pull request merged**